Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -136,7 +171,7 @@ static Boolean __CGPathEqual(CFTypeRef cf1, CFTypeRef cf2) {
 CGMutablePathRef CGPathCreateMutable() {
     __CGPath* mutableRet = __CGPath::CreateInstance();
 
-    mutableRet->initializeGeometries();
+    FAIL_FAST_IF_FAILED(mutableRet->InitializeGeometries());
```

Comment:
>   FAIL_FAST_IF_FAILED(mutableRet->InitializeGeometries()); [](start = 1, length = 59)

Given this is a back and forth discussion we had. This feels more appropriate to return a null, rather than fail fast